pkg/token/token.go

```diff
@@ -65,6 +65,11 @@ type Identity struct {
 	// users or other roles are allowed to assume the role, they can provide
 	// (nearly) arbitrary strings here.
 	SessionName string
+
+	// The AWS Access Key ID used to authenticate the request.  This can be used
+	// in conjuction with CloudTrail to determine the identity of the individual
+	// if the individual assumed an IAM role before making the request.
+	AWSAccessKeyID string
```

Comment:
nit: can you shorten this to just `AccessKeyID` here and all other references? Since AWS is in the project name, it seems redundant :)

Reply:
Done!